dist/lang/pt_strings.js

```diff
@@ -15,9 +15,9 @@ i18n.setKeys({
 	"AliasForm.NewAliasForIndex": "Novo apelido para {0}",
 	"AliasForm.DeleteAliasMessage": "digite ''{0}'' para deletar {1}. Não há como voltar atrás",
 	"AnyRequest.DisplayOptions" : "Mostrar Opções",
-	"AnyRequest.AsGraph" : "Mostrar como grafo",
+	"AnyRequest.AsGraph" : "Mostrar como gráfico",
```

Comment:
Changed the word to avoid mistakes. The english word **graph** can be translated to two different portuguese words, **grafo** (graph) and **gráfico** (graph or chart).

I believe that the original meaning is chart, so I changed to **gráfico**.